test/runtime/lb.go

```diff
@@ -210,15 +210,71 @@ var _ = Describe("RuntimeLB", func() {
 
 		status = docker.ContainerExec(
 			"client",
-			"curl -s --fail --connect-timeout 4 http://2.2.2.2:80/public")
-		status.ExpectSuccess("L3 Proxy is not working IPv6")
+			"curl -s --fail --connect-timeout 4 http://[f00d::1:1]:80/public")
+		status.ExpectSuccess("L4 Proxy is not working IPv6")
 
 		By("L3 redirect to L4")
 		status = cilium.ServiceAdd(3, "2.2.2.2:0", []string{
 			fmt.Sprintf("%s:80", httpd1["IPv4"]),
 			fmt.Sprintf("%s:80", httpd2["IPv4"])}, 2)
 		status.ExpectFail("Service created with invalid data")
 	}, 500)
+
+	It("Service recovery on restart", func() {
+		containers("create")
+		cilium.EndpointWaitUntilReady()
+
+		httpd1, err := docker.ContainerInspectNet("httpd1")
+		Expect(err).Should(BeNil())
+
+		httpd2, err := docker.ContainerInspectNet("httpd2")
+		Expect(err).Should(BeNil())
+
+		status := cilium.ServiceAdd(1, "2.2.2.2:80", []string{
```

Comment:
This IP address is hardcoded and used multiple times in this test. Factor it out into a variable, along with the port.